exercise_01/callbacks/onSuccess.js

```diff
@@ -8,3 +8,15 @@
  * output:
  *  - array
  */
+
+/**
+ * It receives a string and returns an array with all words contained in it
+ * @param {String} value
+ * @returns {Array}
+ */
+function onSuccess(value) {
+  const words = value.split(/\s+/);
+  return words;
```

Comment:
Improve Code Readability with Clearer Code Blocks. Fix it!

```suggestion
 
   return words;
```